<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result</class>
 <widget class="QMainWindow" name="Result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Admin/icon.qrc">
    <normaloff>:/icons/icons/backgrounds/hands.png</normaloff>:/icons/icons/backgrounds/hands.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#result_form{
background-color: rgb(48, 78, 151);
border-radius:40px;
margin:70px;
}
QFrame#border_frame
{
background-color:#6fa844 ;
}
QFrame#main_frame{
background-image: url(:/icons/Background2.jpg);

background-position:center;
background-repeat:no-repeat;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="border_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>80</number>
       </property>
       <property name="topMargin">
        <number>80</number>
       </property>
       <property name="rightMargin">
        <number>80</number>
       </property>
       <property name="bottomMargin">
        <number>80</number>
       </property>
       <item>
        <widget class="QFrame" name="main_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="heading_side_bar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <widget class="QLabel" name="RAAS_heading_label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>121</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(48, 78, 151);
font: 63 32pt &quot;Bahnschrift SemiBold&quot;;</string>
             </property>
             <property name="text">
              <string>RAAS</string>
             </property>
            </widget>
            <widget class="QLabel" name="ET_label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>221</width>
               <height>101</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(48, 78, 151);
font: 25 22pt &quot;Bahnschrift Light&quot;;</string>
             </property>
             <property name="text">
              <string>INSTITUTE OF EMERGING TECHNOLOGIES</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QFrame" name="result_form">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>470</width>
              <height>480</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>800</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <widget class="QLabel" name="test_result_heading">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>130</y>
               <width>181</width>
               <height>51</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(241, 241, 241);
font: 63 20pt &quot;Bahnschrift SemiBold&quot;;
margin:5px;</string>
             </property>
             <property name="text">
              <string>TEST RESULT</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="disp_std_name_label">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>190</y>
               <width>281</width>
               <height>39</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(241, 241, 241);

font: 25 18pt &quot;Bahnschrift Light Condensed&quot;;
margin:5px;</string>
             </property>
             <property name="text">
              <string>Dear Student!</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="scored_marks_label">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>260</y>
               <width>201</width>
               <height>41</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(241, 241, 241);
font: 14pt &quot;Bahnschrift Light Condensed&quot;;
margin:5px;</string>
             </property>
             <property name="text">
              <string>You have scored 98% marks.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="exit_app_btn">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>330</y>
               <width>100</width>
               <height>40</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(111, 168, 68);
border-radius: 15px;
color: rgb(241, 241, 241);
font:  bold 10pt &quot;Bahnschrift Light&quot;;margin:5px;
vertical-align: center;
horizontal-align: center;</string>
             </property>
             <property name="text">
              <string>Exit</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="logo_side_bar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignBottom">
              <widget class="QLabel" name="hunar_logo">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>120</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>120</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/icons/icons/backgrounds/Asset 7-100.jpg);
background-repeat: no-repeat; 
 background-position: right;
margin:5px;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Admin/icon.qrc"/>
 </resources>
 <connections/>
</ui>
